credentials/generate-revocation-set.py

```diff
@@ -133,99 +170,119 @@ def main(use_main_net_dcld, use_test_net_dcld, use_main_net_http, use_test_net_h
         crl_signer_authority_key_id = crl_signer_certificate.extensions.get_extension_for_oid(
             x509.OID_AUTHORITY_KEY_IDENTIFIER).value.key_identifier
 
-        paa_certificate = None
+        # Convert CRL Signer AKID to colon separated hex
+        crl_signer_authority_key_id = crl_signer_authority_key_id.hex().upper()
+        crl_signer_authority_key_id = ':'.join([crl_signer_authority_key_id[i:i+2]
+                                               for i in range(0, len(crl_signer_authority_key_id), 2)])
 
-        # TODO: Extract this to a helper function
-        if use_rest:
-            response = requests.get(
-                f"{rest_node_url}/dcl/pki/certificates/{crl_signer_issuer_name}/{crl_signer_authority_key_id}").json()["approvedCertificates"]["certs"][0]
-            paa_certificate = response["pemCert"]
-        else:
-            cmdlist = ['query', 'pki', 'x509-cert', '-u', crl_signer_issuer_name, '-k', crl_signer_authority_key_id]
-            cmdpipe = subprocess.Popen(use_dcld(dcld, production, cmdlist), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            paa_certificate = json.loads(cmdpipe.stdout.read())["approvedCertificates"]["certs"][0]["pemCert"]
+        paa_certificate = dcld_helper.get_paa_cert_for_crl_issuer(crl_signer_issuer_name, crl_signer_authority_key_id)
 
         if paa_certificate is None:
+            print("PAA Certificate not found, continue...")
             continue
 
-        paa_certificate_object = x509.load_pem_x509_certificate(paa_certificate)
+        paa_certificate_object = x509.load_pem_x509_certificate(bytes(paa_certificate, 'utf-8'))
 
+        # This requires cryptography 40.0.0 and this haults on 40.0.0
+        # try:
+        #     crl_signer_certificate.verify_directly_issued_by(paa_certificate_object)
+        # except Exception as e:
+        #     print("CRL Signer Certificate is not issued by PAA Certificate, continue...")
+        #     continue
```

Comment:
Not sure I understand. Are you saying we need to bump the requested module version?